solo/docker/start.sh

```diff
@@ -23,7 +23,8 @@ done
 curl -XPUT http://127.0.0.1:4001/v2/keys/skydns/${SKYDNS_PATH} \
     -d value="{\"host\":\"$HOST_ADDRESS\"}"
 
-skydns $SKYDNS_OPTS &
+skydns $SKYDNS_OPTS -verbose &
```

Comment:
Yea ill remove.

On Thursday, April 7, 2016, Matt Brown <notifications@github.com> wrote:

> In solo/docker/start.sh
> <https://github.com/spotify/helios/pull/900#discussion_r58858049>:
>
> > @@ -23,7 +23,8 @@ done
> >  curl -XPUT http://127.0.0.1:4001/v2/keys/skydns/${SKYDNS_PATH} \
> >      -d value="{\"host\":\"$HOST_ADDRESS\"}"
> >
> > -skydns $SKYDNS_OPTS &
> > +skydns $SKYDNS_OPTS -verbose &
>
> I'm not sure if -verbose should be put in the image. Seems like you could
> run the docker image with -e SKYDNS_OPTS=-verbose at runtime if you
> really wanted verbose logging.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/spotify/helios/pull/900/files/0d1483b079b076c9cfbfb8e39955dafb9cde6ec8#r58858049>
>


-- 
David Xia
Software Engineer
dxia@spotify.com